clang/lib/Sema/SemaDecl.cpp

```diff
@@ -6886,6 +6886,13 @@ static void checkAttributesAfterMerging(Sema &S, NamedDecl &ND) {
     }
   }
 
+  if (HybridPatchableAttr *Attr = ND.getAttr<HybridPatchableAttr>()) {
+    if (!ND.isExternallyVisible()) {
+      S.Diag(Attr->getLocation(),
+             diag::warn_attribute_hybrid_patchable_non_extern);
+      ND.dropAttr<SelectAnyAttr>();
```

Comment:
```suggestion
      ND.dropAttr<HybridPatchableAttr>();
```

Reply:
Good point, it was a copoy&paste typo. Actually, we don't really need it as LLVM part ignores the attribute for static functions itself. I removed that line and left only diagnostics part here to slightly simplify the code.